### PR TITLE
#175636832 setup hound 

### PR DESCRIPTION
**Pull request by tytyne**
#### What does this PR do?
The PR setup hound in project
#### Description of Task to be completed?
- create .hound.yml file
-  paste the link in the readme link
#### How should this be manually tested?
- ` git clone https://github.com/atlp-rwanda/tech-strikers-bn-backend `
- git checkout ch-setup-hound-175636832
- visit Readme file on github
#### Any background context you want to provide?
- no
#### What are the relevant pivotal tracker stories?
[#175636832](https://www.pivotaltracker.com/n/projects/2475704/stories/175636832)
#### Screenshots (if appropriate)
.hound.yml file
hound
The link in Readme file
hound2

readm

### CODE REVIEW

**Review by Duncanwar — APPROVED** [review]

**Review by Duncanwar — COMMENTED** [review]
Fix the Hound Problem.

**Comment by Duncanwar** [comment]
Solve the conflicts and hound problem

File: .eslintrc.json [hunk 1]
```diff
@@ -18,10 +18,6 @@
     "consistent-return": 0,
     "no-param-reassign": 0,
     "comma-dangle": 0,
-    "quotes": [2, "double", "avoid-escape"],
-    "no-unused-vars": 0,
-    "no-console": 0,
-    "func-names": 0,
```
Comment (kipropbrian) [inline .eslintrc.json:24]: Return these lines.

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+fail_on_violations: true
+
+javascript:
+  enabled: false
+esLint:
+  enabled: true 
+  
+
```
Comment (kipropbrian) [inline .hound.yml:8]: Remove single line.

File: package.json [hunk 3]
```diff
@@ -1,48 +1,33 @@
 {
-  "name": "express-authorshaven",
+  "name": "tech-strikers-bn-backend",
   "version": "1.0.0",
-  "description": "A Social platform for the creative at heart",
+  "description": "[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)",
```
Comment (kipropbrian) [inline package.json:4]: Fix the description. Return to what it was.

Comment (kipropbrian) [reply package.json:4]: Replace your package.json with what is on develop

File: src/index.js [hunk 4]
```diff
@@ -1,15 +1,11 @@
...
-  mongoose = require("mongoose");
-require("dotenv").config();
+import express from "express";
+import { urlencoded, json } from "body-parser";
+import session from "express-session";
+import cors from "cors";
+import errorhandler from "errorhandler";
+import { connect, set } from "mongoose";
+
+import "./models/User.js";
```
Comment (kipropbrian) [inline src/index.js:8]: Revert this file to how it was.

**Review by Duncanwar — COMMENTED** [review]
Implement the feedback brian gave you

**Comment by tytyne** [comment]
had some conflicts with develop

**Pull request closed without merging**